rosidl_generator_cpp/resource/msg__struct.hpp.em

```diff
@@ -286,17 +280,17 @@ using @(spec.base_type.type) =
 
 // constants requiring out of line definition
 @[for c in spec.constants]@
-@[  if c.type not in ['byte', 'int8', 'int16', 'int32', 'int64', 'char', 'uint8', 'uint16', 'uint32', 'uint64']]@
 template<typename ContainerAllocator>
```

Comment:
Does the container allocator make sense for the numeric types?

Reply:
I think that this is necessary, the allocator doesn't apply to the member, but the containing object.  If we used `constexpr`, then we could declare and define in a single statement, but otherwise they need to be separate.